dist/index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
```

Comment:
What's the purpose of the dist folder and what other files do you want to put there.

Reply:
The dist folder would hold production-ready code.
Files like CSS and JS would go in eventually but for now, only "index.html" would be there.

Reply:
@daniellamarr, I think the CSS files should be in an asset folder

Reply:
I just realized the `dist` folder should not be created by me but by the webpack config I already set up. The dist folder is created when a build is ran on the app and only if I specify that the target folder should be `dist`